snap/remote/remote.go

```diff
@@ -90,3 +90,25 @@ type Purchase struct {
 	ItemSKU         string `json:"item_sku,omitempty"`
 	PurchaseID      string `json:"purchase_id,omitempty"`
 }
+
+/*
+PurchaseInstruction encapsulates the data that must be sent
+in order to make a purchase from the store.
+*/
```

Comment:
I believe the preferred style for `godoc` comments is to use multiple single-line comments rather than a multi-line comment